cps/kobo.py

```diff
@@ -678,33 +683,42 @@ def HandleInitRequest():
             url_base=host,
             url_port=config.config_port
         )
+        kobo_resources["image_host"] = calibre_web_url
+        kobo_resources["image_url_quality_template"] = unquote(calibre_web_url + url_for("kobo.HandleCoverImageRequest",
+                                                                                         auth_token=kobo_auth.get_auth_token(),
+                                                                                         book_uuid="{ImageId}",
+                                                                                         width="{width}",
+                                                                                         height="{height}",
+                                                                                         _external=True))
+        kobo_resources["image_url_template"] = unquote(calibre_web_url + url_for("kobo.HandleCoverImageRequest",
+                                                                                 auth_token=kobo_auth.get_auth_token(),
+                                                                                 book_uuid="{ImageId}",
+                                                                                 width="{width}",
+                                                                                 height="{height}",
+                                                                                 _external=True))
```

Comment:
The line external=TRUE gave me wrong urls, the hostname was doubled, I removed it